src/main.rs

```diff
@@ -81,31 +81,28 @@ fn enclosure_overview(option: &ArgMatches) -> Result<(), ()> {
                 if enc.slot == disk.enclosure {
                     print!("     `+-");
                     print!(" Disk: {:<10}", disk.device_path.green(),);
-                    if disk.device_map == "NONE" {
-                        print!(" Map: {:<10}", disk.device_map.yellow());
+                    let device_map_color = if disk.device_map == "NONE" {
+                        disk.device_map.yellow()
                     } else {
-                        print!(" Map: {:<10}", disk.device_map.green());
-                    }
+                        disk.device_map.green()
+                    };
```

Comment:
if-expr are neats, produce quite good assembly and nice to read it comes from Ocaml/Haskell/Scala inspiration.